### PR TITLE
Upgrade form lib

### PR DESCRIPTION
**Pull request by vcapretz**
## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/components/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
+import React, { Component } from 'react'
+import {
+  node,
+  object,
```
Comment (augusto-jm-amaral) [inline src/components/Form/index.js]: Acho que não desestruturamos os propTypes.

File: src/components/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+      </form>
+    )
+  }
+}
+
+Form.propTypes = {
+  children: node,
+  className: string,
+  customErrorProp: string,
+  data: object, // eslint-disable-line
```
Comment (augusto-jm-amaral) [inline src/components/Form/index.js]: Não conseguimos trocar por ´propTypes.shape()´?

Comment (vcapretz) [reply src/components/Form/index.js]: aqui não @augusto-jm-amaral! o `data` é o que o form vai receber, e pode ser qualquer combinação de nomes de input sabe?

File: src/components/Form/index.js (hunk 1 above)
Comment (vcapretz) [reply src/components/Form/index.js]: mudando

**Pull request merged**